.gitignore

```diff
@@ -1,18 +1,19 @@
 *.exe
 *.exe.stackdump
 *.gcda
+*.gch
```

Comment:
Can you add a Signed-off-by: to your patch?

Reply:
Done; sorry about that!